apps/sscs/aat/01/kustomization.yaml

```diff
@@ -2,4 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../base
+  - ../../sscs-case-loader/sscs-case-loader.yaml
```

Comment:
Do we not want to include this in aat base rather than 00 & 01?

Reply:
scripting is easy this way :D Problem is not all apps have them on both clusters.

Reply:
I can try and find a way, if u think it will be cleaner.

Reply:
Cheers! Sorry just saw the duplicate of that base reference^